en/Pepper-and-Carrot_by-David-Revoy_E23P04.svg

```diff
@@ -147,12 +147,12 @@
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="283.55594"
-         id="tspan470">public holidays. Offer is</tspan><tspan
+         id="tspan470">public holidays. Offer may</tspan><tspan
          sodipodi:role="line"
          style="font-style:italic;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:35.77793884px;line-height:1.25;font-family:Lavi;-inkscape-font-specification:'Lavi Italic'"
          x="1856.9244"
          y="328.27838"
-         id="tspan472">not cumulative. Limited to</tspan><tspan
```

Comment:
"may not be combined" is what every USA folk sees on their coupons (paper discounts offered by establishments and marketers in order to entice folks to purchase their wares). So "may not be combined with any other offer" will ring closer to what someone in the USA would see and associate with legal "blah blah blah" along with "offer void where prohibited".